test/torchaudio_unittest/example/tacotron2/test_text_preprocessing.py

```diff
@@ -1,11 +1,10 @@
-import unittest
-
 from parameterized import parameterized
 
-from .text_preprocessing import text_to_sequence
+from .text.text_preprocessing import text_to_sequence
+from torchaudio_unittest.common_utils import PytorchTestCase
 
 
-class TestTextPreprocessor(unittest.TestCase):
+class TestTextPreprocessor(PytorchTestCase):
```

Comment:
nit: `TorchaudioTestCase` is slightly preferred as it will prevent accidental use of I/O functions. (which is not the case for text utils)